spring-boot-docs/src/main/asciidoc/using-spring-boot.adoc

```diff
@@ -892,6 +892,9 @@ devtools support, simply add the module dependency to your build:
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-devtools</artifactId>
 			<optional>true</optional>
+			<configuration>
+                    		<fork>true</fork>
+                        </configuration>
 		</dependency>
```

Comment:
Seems like you got my idea and have one yourself. I would appreciate if you could go on then.
Thanks.